internal/service/api/requests/get_role.go

```diff
@@ -14,6 +15,15 @@ func NewGetRoleRequest(r *http.Request) (GetRoleRequest, error) {
 	var request GetRoleRequest
 
 	err := urlval.Decode(r.URL.Query(), &request)
+	if err != nil {
+		return request, err
+	}
 
-	return request, err
+	return request, validateGetRoleRequest(request)
+}
+
+func validateGetRoleRequest(request GetRoleRequest) error {
+	return validation.Errors{
+		"accessLevel": validation.Validate(request.AccessLevel, validation.Required),
```

Comment:
better to specify access levels that can be sent, using e.g. validation.In("admin", "user" etc.)